sdk/appconfiguration/app-configuration/test/public/index.spec.ts

```diff
@@ -19,24 +19,23 @@ import {
   startRecorder,
   toSortedArray,
   toSortedLabelsArray,
-} from "./utils/testHelpers";
-import { Context } from "mocha";
-import { assert } from "chai";
+} from "./utils/testHelpers.js";
+import { describe, it, assert, beforeEach, afterEach, afterAll } from "vitest";
 
 describe("AppConfigurationClient", () => {
   let client: AppConfigurationClient;
   let recorder: Recorder;
 
-  beforeEach(async function (this: Context) {
-    recorder = await startRecorder(this);
+  beforeEach(async function (ctx) {
+    recorder = await startRecorder(ctx);
     client = createAppConfigurationClientForTests(recorder.configureClientOptions({}));
   });
 
-  afterEach(async function (this: Context) {
+  afterEach(async function () {
     await recorder.stop();
   });
 
-  after(async function (this: Context) {
+  afterAll(async function () {
     if (!isPlaybackMode()) {
       await deleteEverySetting();
```

Comment:
Depending on how long this takes you might see a hook timeout and need to add `--hook-timeout <somevalue>` for it

Reply:
The live test pipelines seem to work with the default value, so I'll keep the default for now